insights/parsers/x86_debug_enabled.py

```diff
@@ -0,0 +1,33 @@
+"""
+Parsers for file ``/sys/kernel/debug/x86/*_enabled`` outputs
+============================================================
+
+This module provides the following parsers:
+
+X86Enabled - file ``/sys/kernel/debug/x86/*_enabled``
+------------------------------------------------------------
```

Comment:
I believe it should be `X86DebugEnabled`

BTW, keep the underline '---' and title in the same length.